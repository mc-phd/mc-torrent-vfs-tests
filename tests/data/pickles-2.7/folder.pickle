(dp1
S'creation date'
p2
I1449730049429
sS'info'
p3
(dp4
S'files'
p5
(lp6
(dp7
S'path'
p8
(lp9
S'file.txt'
p10
asS'length'
p11
I15
sasS'piece length'
p12
I16384
sS'name'
p13
S'folder'
p14
sS'pieces'
p15
S'y\x9c\x11\xe3H\xd3\x9f\x17\x04\x02+\x83TP./\x81\xf3\xc07'
p16
ssS'encoding'
p17
S'UTF-8'
p18
s.