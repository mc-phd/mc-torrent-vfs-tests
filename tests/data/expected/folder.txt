dr-xr-xr-x 1 user group 0 02-12-9999 23:30 .META
-r--r--r-- 1 user group 25 02-12-9999 23:30 .META/creation date
-r--r--r-- 1 user group 5 02-12-9999 23:30 .META/encoding
-r--r--r-- 1 user group 5 02-12-9999 23:30 .META/piece length
dr-xr-xr-x 1 user group 0 02-12-9999 23:30 folder
-r--r--r-- 1 user group 15 02-12-9999 23:30 folder/file.txt
